packages/zoe/test/unitTests/contracts/test-secondPriceAuction.js

```diff
@@ -293,27 +304,45 @@ test('zoe - secondPriceAuction w/ 3 bids - alice exits onDemand', async t => {
     // Bob bids
     const bobSeat = await zoe.offer(bobInvitation, bobProposal, bobPayments);
 
-    t.rejects(
-      () => E(bobSeat).getOfferResult(),
-      /The item up for auction is not available or the auction has completed/,
-      'The bid should have failed.',
+    // Bob exits early, but this does not stop the auction.
+    bobSeat.tryExit();
+
+    const carolInvitation = await E(makeInvitationObj).makeBidInvitation();
+
+    const carolProposal = harden({
+      want: { Asset: moola(1) },
+      give: { Bid: simoleans(8) },
+    });
+    const carolPayments = harden({ Bid: carolSimoleanPayment });
+
+    // Carol bids
+    const carolSeat = await zoe.offer(
+      carolInvitation,
+      carolProposal,
+      carolPayments,
     );
 
+    timer.tick();
+
     const aliceMoolaPayout = await aliceSeat.getPayout('Asset');
     const aliceSimoleanPayout = await aliceSeat.getPayout('Ask');
     const bobMoolaPayout = await bobSeat.getPayout('Asset');
     const bobSimoleanPayout = await bobSeat.getPayout('Bid');
+    const carolMoolaPayout = await carolSeat.getPayout('Asset');
+    const carolSimoleanPayout = await carolSeat.getPayout('Bid');
 
-    // Alice (the creator of the auction) gets back what she put in
+    // Alice (the creator of the auction) gets Carol's simoleans and
+    // carol gets the assets.
     t.deepEquals(
       await moolaR.issuer.getAmountOf(aliceMoolaPayout),
-      aliceProposal.give.Asset,
+      moola(0),
+      `alice has no assets`,
```

Comment:
Alice has no moola

Reply:
I think this is the same thing

Reply:
alice has simoleans, so I was objecting to a line in the test transcript that shows that 'alice has no assets'. It should say 'alice has no moola'.

Reply:
Ah, I mean the keyword 'Asset' - she has none of these, which are moola. I'll change this for clarity.

Reply:
Done!

Reply:
ah, I see. She has assets, but no Assets. :-)